openshift/e2e-common.sh

```diff
@@ -191,6 +191,29 @@ function run_e2e_tests(){
   local test_name=$1 
   local failed=0
 
+  # Keep this in sync with test/ha/ha.go
+  readonly REPLICAS=2
+  readonly BUCKETS=10
+
+  # Keep the bucket count in sync with test/ha/ha.go
+  # TODO: configure it in KnativeServing when operator supports it.
+  kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-leader-election --type=merge \
+    --patch='{"data":{"buckets": "'${BUCKETS}'"}}' || failed=1
```

Comment:
I'm curious if this logic should be added a little later, right before we want to run the "ha" tests.

Reply:
Yes, either should be fine. But I tried to change it same as upstream does.

https://github.com/knative/serving/blob/e2a99ff93cbeca6162fbb91f1ab6eb11c79d0326/test/e2e-tests.sh#L79

Perhaps, it is better to run e2e and conformance tests with enabling buckets to confirm changing the buckets size does not effect on them.